src/pages/folder/Folder.jsx

```diff
@@ -0,0 +1,91 @@
+import { useEffect } from 'react';
+import Header from '../../components/Header';
+import Card from '../../components/Card';
+import Input from '../../components/Input';
+import Footer from '../../components/Footer';
+import FolderInfo from '../../components/FolderInfo';
+
+import styled from 'styled-components';
+import useGetData from '../../components/hooks/useGetData';
+
+const MainContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 1060px;
+  margin: auto;
+
+  @media screen and (max-width: 1023px) {
+    width: auto;
+    margin-left: 32px;
+    margin-right: 32px;
+  }
+
+  @media screen and (max-width: 767px) {
+    margin-right: 32px;
+    margin-left: 32px;
+  }
+`;
+
+const CardContainer = styled.div`
+  display: grid;
+
+  grid-template-columns: repeat(3, 1fr);
+  column-gap: 2rem;
+  row-gap: 2rem;
+  justify-content: center;
+
+  @media screen and (max-width: 1023px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media screen and (max-width: 767px) {
+    grid-template-columns: 1fr;
+  }
+`;
+
+const BASE_URL = 'https://bootcamp-api.codeit.kr/api/sample/folder';
+
+function Folder() {
+  const [data, isLoading, error, fetchData] = useGetData(BASE_URL);
+
+  useEffect(() => {
+    fetchData(BASE_URL);
+  }, [BASE_URL]);
```

Comment:
BASE_URL 은 상태값이 아닌 상수값이기 때문에 dependency 배열에 넣어도 안넣은 것과 동일하게 맨 처음 Folder 가 렌더링될 때만 실행됩니다! 제거해주세요!
```suggestion
  }, []);
```

Reply:
상수는 변하지 않으니까, dependency 배열에 안넣어도 되는군요! 
수정했습니다 감사합니다! 👍